README.md

```diff
@@ -1,2 +1,15 @@
-# CircleCI-HockeyApp-Appium-ReactNative
-Guidance on integrating Kobiton service into the mobile app build pipeline: CircleCI, HockeyApp, Appium and ReactNative
+# Integrating Kobiton into CircleCI mobile application development pipeline
+
+In application development, especially mobile application development, perform testing on **real** devices before releasing is a crucial, but also costly process, which means that only few developers are able to perform testing on real devices, but only a very limited range of devices.
+
+By integrating Kobiton into your mobile application development pipeline, CircleCI will able to execute an automation testing on real devices, which help improve development process and product quality.
+
+This guide is applicable if you are using these tools and technologies in your pipeline:
+- ReactNative as a technology to develop the application with.
+- Appium as a framework to develop the test scripts using.
+- CircleCI as a CI/CD tool.
+- GitHub as a VCS provider to store your test scripts on.
+
+<img src="docs/assets/diagram.svg"/>
```

Comment:
@NguyenHoanTien  Put the link of "source code" of diagram in the jira description